climada/entity/impact_funcs/trop_cyclone.py

```diff
@@ -22,18 +22,317 @@
 __all__ = ["ImpfTropCyclone", "ImpfSetTropCyclone", "IFTropCyclone"]
 
 import logging
+from enum import Enum
 
 import numpy as np
 import pandas as pd
 from deprecation import deprecated
 
 from climada.entity.impact_funcs.base import ImpactFunc
 from climada.entity.impact_funcs.impact_func_set import ImpactFuncSet
+from climada.util import coordinates
 from climada.util.constants import SYSTEM_DIR
 
 LOGGER = logging.getLogger(__name__)
 
 
+class CountryCode(Enum):
+    """
+    Enum class that links ISO country codes (both ISO3A and ISO3N) to specific regions and
+    associated impact function IDs.
+
+    Attributes
+    ----------
+        ISO3A: dict
+             A mapping of region names to lists of 3-letter ISO country codes (iso3a).
+        ISO3N: dict
+             A mapping of region names to lists of numeric ISO country codes (iso3n).
+        IMPF_ID: dict
+            A mapping of region names to corresponding impact function IDs.
+        REGION_NAME: dict
+            A mapping of region names to their descriptive names.
+    """
+
+    ALPHA3 = {
+        "NA1": [
```

Comment:
what about putting these lists within a `# fmt: off` `# fmt: on` block? You could have lines with, e.g,, 10 country codes. 😁